.pre-commit-config.yaml

```diff
@@ -1254,93 +1252,93 @@ repos:
         entry: ./scripts/ci/pre_commit/mypy.py
         files: ^dev/.*\.py$|^scripts/.*\.py$
         require_serial: true
-        additional_dependencies: ['rich>=12.4.4']
+        additional_dependencies: ["rich>=12.4.4"]
       - id: mypy-dev
-        stages: [ 'manual' ]
+        stages: ["manual"]
         name: Run mypy for dev (manual)
         language: python
         entry: ./scripts/ci/pre_commit/mypy_folder.py dev
         pass_filenames: false
         files: ^.*\.py$
         require_serial: true
-        additional_dependencies: [ 'rich>=12.4.4' ]
+        additional_dependencies: ["rich>=12.4.4"]
       - id: mypy-airflow
         name: Run mypy for airflow
         language: python
         entry: ./scripts/ci/pre_commit/mypy.py --namespace-packages
         files: \.py$
         exclude: ^.*/.*_vendor/|^airflow/migrations|^airflow/providers|^dev|^scripts|^docs|^provider_packages|^tests/providers|^tests/system/providers|^tests/dags/test_imports.py|^clients/python/test_.*\.py
         require_serial: true
-        additional_dependencies: ['rich>=12.4.4']
+        additional_dependencies: ["rich>=12.4.4"]
       - id: mypy-airflow
-        stages: [ 'manual' ]
+        stages: ["manual"]
         name: Run mypy for airflow (manual)
         language: python
         entry: ./scripts/ci/pre_commit/mypy_folder.py airflow
         pass_filenames: false
         files: ^.*\.py$
         require_serial: true
-        additional_dependencies: [ 'rich>=12.4.4' ]
+        additional_dependencies: ["rich>=12.4.4"]
       - id: mypy-providers
         name: Run mypy for providers
         language: python
         entry: ./scripts/ci/pre_commit/mypy.py --namespace-packages
         files: ^airflow/providers/.*\.py$|^tests/providers/.*\.py$|^tests/system/providers/.*\.py$
         exclude: ^.*/.*_vendor/
         require_serial: true
-        additional_dependencies: ['rich>=12.4.4']
+        additional_dependencies: ["rich>=12.4.4"]
       - id: mypy-providers
-        stages: ['manual']
+        stages: ["manual"]
         name: Run mypy for providers (manual)
         language: python
         entry: ./scripts/ci/pre_commit/mypy_folder.py airflow/providers
         pass_filenames: false
         files: ^.*\.py$
         require_serial: true
-        additional_dependencies: ['rich>=12.4.4']
+        additional_dependencies: ["rich>=12.4.4"]
```

Comment:
Some vscode / IDE formatter ran on this file. The original form was intended. Can we revert the formatting part ?

Reply:
Nice catch! This file's format was updated automatically when I edited it with the new settings to exclude `pnpm-lock.yaml` from the "Add license" script. I'll change the config to only be active on `airflow/ui` ignoring everything else. I'll do that first thing in the morning (it's 3:30 AM for me currently, so in a few hours 😅).

Reply:
Fixed 😄